lib/loader/index.d.ts

```diff
@@ -5,9 +5,13 @@ export interface ResultObject {
   instance: WebAssembly.Instance;
 }
 
-/** WebAssembly imports with two levels of nesting. */
-export type Imports = {
-  [key: string]: object,
+/** Helper type for WebAssembly imports that defines two levels of nesting. */
+type ImportsHelper = {
+  [key: string]: any;
```

Comment:
How does this differ from just changing `object` to `any` below?

Reply:
It differes because when I want to use the typescript types, I get a tsc compiler error that the `env` object type is not assignable to `object`.

This makes the types more correct and explicit.

Reply:
Technically the type should be `WebAssembly.Memory | WebAssembly.Table | number | string | (...args: number[]) => void | number`

Is that something worth changing? Or could it just be `any`? As far as I can tell, you can use `anyref` imports, so it really could be `any`.

Reply:
Do you think it would be better to just make it `any`?

Reply:
My question was rather about how using `ImportsHelper` differs from changing the `[key: string]: object` index signature below to the equivalent without adding a helper class.

Regarding the type: Not sure what's best, but making things too restrictive leads to problems when specifying the `binaryen` object from binaryen.js as an import for example, as it does not fulfill the criteria.

Reply:
I switched to an inline `[key: string]: any` in the current pull request. Does this satisfy your requirements?

Reply:
And honestly, I'm not sure I understand the binaryen problem. All I can replicate locally is the typescript error that throws every time I want to use the loader in typescript.

Reply:
I think you are misunderstanding. Was just curious if there is a difference :)

Reply:
Regarding the Binaryen problem: I noticed this in `tests/bootstrap/index.ts`, which uses `{ binaryen }` as the imports object. This didn't type-check before I made the changes to `[key: string]: object`, but then lead to the errors your PR now tries to fix. All kinda unfortunate.

Reply:
There isn't a difference, not effectively anyway. I often like to separate out my types so that I can avoid operator precedence problems. In this case, the second commit I applied to this pull request is exactly equivalent, and accurately represents what an import can be: `any`.